pkg/exposer/csi_snapshot.go

```diff
@@ -262,7 +261,13 @@ func (e *csiSnapshotExposer) CleanUp(ctx context.Context, ownerObject corev1.Obj
 	backupVSName := ownerObject.Name
 
 	kube.DeletePodIfAny(ctx, e.kubeClient.CoreV1(), backupPodName, ownerObject.Namespace, e.log)
-	kube.DeletePVCIfAny(ctx, e.kubeClient.CoreV1(), backupPVCName, ownerObject.Namespace, e.log)
+	pvc, err := e.kubeClient.CoreV1().PersistentVolumeClaims(ownerObject.Namespace).Get(ctx, backupPVCName, metav1.GetOptions{})
```

Comment:
Let's put the PVC existence detection and get PV action into `DeletePVAndPVCIfAny` function, so that we don't need to change every places calling `DeletePVAndPVCIfAny`

Reply:
because in same other places it could get PVC directly, but in CleanUp it could not, so in order to an unnecessary Call  Get PVC, so I move it out of `DeletePVAndPVCIfAny`